package.json

```diff
@@ -117,6 +117,7 @@
     "enhanced-resolve": "^3.4.1",
     "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.11.2",
+    "has-flag": "^2.0.0",
```

Comment:
please don't add any additional libraries. if there's a need please consult with the whole team.